packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -122,13 +122,30 @@ export type VisibilityState = (typeof VisibilityState)[keyof typeof VisibilitySt
  * @sealed
  */
 export interface IContainerRuntimeBaseEvents extends IEvent {
-	(event: "batchBegin", listener: (op: ISequencedDocumentMessage) => void);
-	/**
-	 * @param runtimeMessage - tells if op is runtime op. If it is, it was unpacked, i.e. it's type and content
+	(
+		event: "batchBegin",
+		listener: (
+			op: Omit<ISequencedDocumentMessage, "contents"> & {
```

Comment:
Isn't it breaking since we are Omitting the contents param in this PR. @Josmithr